lib/remote/apilistener.cpp

```diff
@@ -1144,7 +1144,8 @@ void ApiListener::ReplayLog(const JsonRpcConnection::Ptr& client)
 				}
 
 				try  {
-					NetString::WriteStringToStream(client->GetStream(), pmessage->Get("message"));
+					size_t bytesSend = NetString::WriteStringToStream(client->GetStream(), pmessage->Get("message"));
+					endpoint->AddMessageSent(bytesSend);
```

Comment:
I'm not happy with the naming schema. 

`message sent` does not directly tell you what it does. `bytesSend` should probably renamed to `sentBytes`.

This likely affects multiple code locations throughout.

Reply:
I already used bytesSent in other files, so I'll correct it to that.